src/server/main_service.cc

```diff
@@ -617,6 +617,10 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
     return true;
   });
 
+  config_registry.Register("dir", [](const absl::CommandLineFlag& flag) { return true; });
```

Comment:
Perhaps we could add some default function that does nothing, or an overload that does so?